### PR TITLE
Added logtest example on devtools.

### PR DESCRIPTION
**Pull request by gabiwassan**
Hi team,
This PR added a PUT logtest example on devtools.

image

Closes: #3061

### CODE REVIEW

**Review by asteriscos — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,13 @@ All notable changes to the Wazuh app project will be documented in this file.
...
 
+### Added
+
+- Add logtest PUT example on API Console [#3061](https://github.com/wazuh/wazuh-kibana-app/pull/3061)
+
+### Fixed
 
+- Fix wrong variable declaration for macOS agents [#3066](https://github.com/wazuh/wazuh-kibana-app/pull/3066)
 
 ## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4104
```
Comment (Desvelao) [inline CHANGELOG.md:15]: Why has the `CHANGELOG.md`  file 2 not related changes?

Comment (gabiwassan) [reply CHANGELOG.md:15]: Mmmm maybe when I applied rebase... let me check

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by pablomarga — APPROVED** [review]
LGTM!

**Pull request merged**